hail/src/main/scala/is/hail/expr/ir/TableIR.scala

```diff
@@ -531,19 +534,18 @@ case class PartitionNativeReader(spec: AbstractTypedCodecSpec) extends AbstractN
       val decRes = mb.newEmitLocal(PInt64Optional)
       val hasNext = mb.newLocal[Boolean]("pnr_hasNext")
       val next = mb.newLocal[Long]("pnr_next")
-      val stream = Stream.unfold[Code[Long]](
+
+      SizedStream.unsized(eltRegion => Stream.unfold[Code[Long]](
```

Comment:
These diffs look weird, but what I did was move the `val stream` definition inside the `unsized` call and use `eltRegion` instead of `region`.